src/script/components/Modals/PrimaryModal/PrimaryModalTypes.ts

```diff
@@ -68,18 +74,20 @@ export enum PrimaryModalType {
 
 export interface ModalContent {
   checkboxLabel: string;
+  closeBtnTitle?: string;
   closeFn: () => void;
   closeOnConfirm?: boolean;
+  copyPassword?: boolean;
   currentType: string;
   inputPlaceholder: string;
   messageHtml: string;
-  messageText: string;
+  messageText: React.ReactNode;
```

Comment:
maybe we could call this one `message` and deprecate the `messageHtml` (since react node is way more suitable for the future)

Reply:
ive updated this and the other modal comment in [1d4a248](https://github.com/wireapp/wire-webapp/pull/15014/commits/1d4a2486629eb44cf74581c4681632b28a7a3a07) but my sense is that these are sort of unrelated to the main PR here.